ENH: Add saveSequenceChanges property to meta reader

nathanbmnt opened the pull request:
When adding multiple volume sequences to the same browser, if the timestamps of the sequences are not exactly the same, duplicate frames will be created at the master sequence node's timestamps because SetSaveChanges is set to true.

For example if load a .seq.mha with 3 frames. Then load another .seq.mha with 3 frames into the same browser. Checking the number of frames in the second sequence with `.GetNumberOfDataNodes()`, returns 4.

This PR adds the loading property saveSequenceChanges, which is true by default.
![image](https://user-images.githubusercontent.com/40271210/151586351-7c5a29be-2599-46cf-98f9-d8335a323fc7.png)
![image](https://user-images.githubusercontent.com/40271210/149798734-b5d56285-c077-4aae-8997-3646e60f5cb0.png)

nathanbmnt commented:
@Sunderlandkyl @lassoan What are your opinions on this change?

Sunderlandkyl commented:
I haven't tested it, but based on the comment, I would think that this change could significantly slow down playback for sequences loaded from Metafile. Is there a noticeable slowdown?

@lassoan is this still a consideration with the latest Sequences?

nathanbmnt commented:
@Sunderlandkyl it seems that Andras is preoccupied. I could make this PR a loading property? Like `saveSequenceChanges` with default of True.

Sunderlandkyl commented:
Sorry, I was preoccupied as well. Sure, that sounds good to me.

nathanbmnt committed:
ENH: Add saveSequenceChanges property to meta reader

Having save sequence changes true can cause duplicate frames to be created in sequence nodes, so allow it to be false.

nathanbmnt commented:
@Sunderlandkyl @lassoan I've changed this PR so it adds the property saveSequenceChanges, which is true by default.

nathanbmnt commented:
Pinging @Sunderlandkyl @lassoan

Sunderlandkyl commented:
Thanks! LGTM!
